### ISSUE TITLE
Direct links to Showcase leading to site select screen AND not opening

### ISSUE THREAD

**Issue by ianstewart** [Automattic/wp-calypso#71033]
![image](https://user-images.githubusercontent.com/545779/206871655-d4a3e8a0-41ee-4055-81df-bacb32e6e3ea.png)

Direct links to the theme showcase for logged-in users are leading to the select site screen. Creating a frustrating experience and maybe a bug.

Frustrating experience:

Ideally this would lead to your primary blog or last used blog or at least a nicer screen.

Maybe a bug:

I wasn't able to actually switch sites on this screen. (Note: I have a lot of test sites and this may be the actual issue.)

**Comment by autumnfjeld** [comment]
<img width="469" alt="image" src="https://user-images.githubusercontent.com/5665959/207913750-4d4835bb-7fe1-4e78-bea7-8753ac9d878e.png">

**Comment by Copons** [comment]
> I wasn't able to actually switch sites on this screen.

We have confirmed this.

I have increased the priority of this issue on the team board to high to fix this bug.

> Frustrating experience: Ideally this would lead to your primary blog or last used blog or at least a nicer screen.

This is less important IMHO. Landing on the site selector is not particularly exciting, but also not broken.

This said, I'm pretty sure some of the other sections perform a "pre-selection". We might want to give a quick look at the various routings and adopt an existing behaviour.

I don't have a strong opinion between the three options here:
- Keep the site selection (current).
- Pre-select the primary site.
- Pre-select the last used site.
